pkg/webhooks/handlers.go

```diff
@@ -42,26 +42,14 @@ func errorResponse(logger logr.Logger, err error, message string) *admissionv1.A
 	return admissionutils.ResponseFailure(false, message+": "+err.Error())
 }
 
-func setupLogger(logger logr.Logger, name string, request *admissionv1.AdmissionRequest) logr.Logger {
-	return logger.WithName(name).WithValues(
-		"uid", request.UID,
-		"kind", request.Kind,
-		"namespace", request.Namespace,
-		"name", request.Name,
-		"operation", request.Operation,
-		"gvk", request.Kind.String(),
-	)
-}
```

Comment:
Is it possible to include resource meta in the logger? They are helpful in debugging.